lib/ai_bot/tools/tool.rb

```diff
@@ -77,8 +77,34 @@ def standalone?
 
         protected
 
-        def send_http_request(url, headers: {}, authenticate_github: false)
-          uri = URI(url)
+        def send_http_request(url, headers: {}, authenticate_github: false, follow_redirects: false)
```

Comment:
Still feels really clunky to have `authenticate_github` as a top-level keyword arg here, IMO as I was saying in the previous review the github tools should just pass in the `Authorization` header themselves as part of `headers`

Reply:
yeah but Its so much ceremony in 3 tools, kind of saves a bunch for them I hear you though.

Reply:
oh ... we are cooking as admin so we get follows :( will add nofollows to a few places ...